recipes/sos-notebook/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "sos-notebook" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9f3ae4371f2a9cf493c77190010e73270ddc658904618937889ce14c0ecebb09
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ PYTHON }} -m sos_notebook.install --sys-prefix
+
+requirements:
+  host:
+    - ipython
+    - notebook >=5.0.0
+    - pip
+    - python >=3.6
+  run:
+    - ipykernel
+    - ipython
+    - markdown
+    - matplotlib
+    - nbconvert >=5.1.1
+    - nbformat
+    - notebook >=5.0.0
+    - pandas
+    - python >=3.6
+    - setuptools
+    - sos >=0.19.2
```

Comment:
I have no idea why inclusion of this file will lead to

```
  File "/Users/bpeng1/anaconda3/lib/python3.6/site-packages/conda/resolve.py", line 244, in verify_specs
    raise ResolvePackageNotFound(bad_deps)
conda.exceptions.ResolvePackageNotFound: 
  - sos[version='>=0.19.2']
```

Reply:
It is likely that some dependency of `sos` conflicts with the dependency of `sos-notebook`, but checking [here](https://github.com/conda-forge/sos-feedstock/blob/master/recipe/meta.yaml) I do not see anything suspicious, and the error message `"ResolvePakcageNotFound"` does not show anything specific.